packages/bigtable/src/index.js

```diff
@@ -262,267 +276,263 @@ function Bigtable(options) {
+ *     .on('complete', function() {
+ *       // The instance has successfully been created.
+ *     });
  * };
  *
- * bigtable.createTable('prezzy', options, callback);
- *
- * //-
- * // You can also specify garbage collection rules for your column families.
- * // See {module:bigtable/table#createFamily} for more information about
- * // column families and garbage collection rules.
- * //-
  * var options = {
- *   families: [
+ *   displayName: 'my-sweet-instance',
+ *   clusters: [
  *     {
- *       name: 'follows',
- *       rule:  {
- *         age: {
- *           seconds: 0,
- *           nanos: 5000
- *         },
- *         versions: 3,
- *         union: true
- *       }
+ *       name: 'my-sweet-cluster',
+ *       nodes: 3,
+ *       location: 'us-central1-b',
+ *       storage: 'ssd'
  *     }
  *   ]
  * };
  *
- * bigtable.createTable('prezzy', options, callback);
- *
- * //-
- * // Pre-split the table based on the row key to spread the load across
- * // multiple Cloud Bigtable nodes.
- * //-
- * var options = {
- *   splits: ['10', '20']
- * };
- *
- * bigtable.createTable('prezzy', options, callback);
+ * bigtable.createInstance('my-instance', options, callback);
  */
-Bigtable.prototype.createTable = function(name, options, callback) {
+Bigtable.prototype.createInstance = function(name, options, callback) {
   var self = this;
 
-  options = options || {};
-
   if (is.function(options)) {
     callback = options;
     options = {};
   }
 
-  if (!name) {
-    throw new Error('A name is required to create a table.');
-  }
-
   var protoOpts = {
-    service: 'BigtableTableService',
-    method: 'createTable'
+    baseUrl: this.adminBaseUrl_,
+    service: 'BigtableInstanceAdmin',
+    method: 'createInstance'
   };
 
   var reqOpts = {
-    name: this.clusterName,
-    tableId: name,
-    table: {
-      // The granularity at which timestamps are stored in the table.
-      // Currently only milliseconds is supported, so it's not configurable.
-      granularity: 0
+    parent: this.projectName,
+    instanceId: name,
+    instance: {
+      displayName: options.displayName || name
     }
   };
 
-  if (options.operation) {
-    reqOpts.table.currentOperation = options.operation;
-  }
-
-  if (options.splits) {
-    reqOpts.initialSplitKeys = options.splits;
-  }
-
-  if (options.families) {
-    var columnFamilies = options.families.reduce(function(families, family) {
-      if (is.string(family)) {
-        family = {
-          name: family
-        };
-      }
-
-      var columnFamily = families[family.name] = {};
-
-      if (is.string(family.rule)) {
-        columnFamily.gcExpression = family.rule;
-      } else if (is.object(family.rule)) {
-        columnFamily.gcRule = Family.formatRule_(family.rule);
-      }
+  reqOpts.clusters = arrify(options.clusters)
+    .reduce(function(clusters, cluster) {
+      clusters[cluster.name] = {
+        location: reqOpts.parent + '/locations/' + cluster.location,
```

Comment:
Should we detect if the user provided a `location` path already formed like you're doing here?